src/main/java/org/apache/flink/benchmark/UnalignedCheckpointTimeBenchmark.java

```diff
@@ -116,6 +116,7 @@ protected int getNumberOfSlotsPerTaskManager() {
 
         @Override
         protected int getNumberOfTaskManagers() {
+            // why is this using PARALLELISM when we don't actually use it?
```

Comment:
this is probably a bug, `PARALLELISM` should have been used instead of `parallelism` in `env.setParallelism(parallelism);`. But let's keep it for now for continuity of the benchmark results while we are merging this? I can fix it later on.